pkg/eventcache/eventcache.go

```diff
@@ -141,7 +140,10 @@ func (ec *Cache) handleEvents() {
 				tmp = append(tmp, event)
 				continue
 			}
-			if errors.Is(err, ErrFailedToGetProcessInfo) {
+			if errors.Is(err, ErrFailedToGetParentInfo) {
+				errormetrics.ErrorTotalInc(errormetrics.EventCacheParentInfoFailed)
+			} else if errors.Is(err, ErrFailedToGetProcessInfo) {
+				errormetrics.ErrorTotalInc(errormetrics.EventCacheProcessInfoFailed)
```

Comment:
We can delete `EventCacheParentInfoFailed` and `EventCacheProcessInfoFailed` entirely from `errormetrics` I think, and define `ParentInfoErrors` in `eventcachemetrics`, similar to `ProcessInfoError`. The goal is to 1) avoid duplicate metrics (like it seems to be the case now for process info errors) 2) report different errors in a consistent way.